From ba4f9dd74ccb9da91195b3570310754716064ef2 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Tue, 10 Oct 2017 15:55:31 +0800
Subject: [PATCH] Revert "usb: kconfig: remove dependency FSL_SOC for ehci fsl
 driver"

This reverts commit 92042e8b3622a9bbfce0ebfc90edf6cd14d45708 on
LSDK linux (https://github.com/qoriq-open-source/linux).

The patch reverted allowed to build ehci-fsl driver for non-PPC
platforms, but actually the driver was not ready.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 drivers/usb/host/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -165,7 +165,7 @@ config XPS_USB_HCD_XILINX
 
 config USB_EHCI_FSL
 	tristate "Support for Freescale PPC on-chip EHCI USB controller"
-	depends on USB_EHCI_HCD
+	depends on FSL_SOC
 	select USB_EHCI_ROOT_HUB_TT
 	---help---
 	  Variation of ARC USB block used in some Freescale chips.
